whois/plugin.spec.yaml

```diff
@@ -60,7 +61,7 @@ actions:
         type: date
         required: false
       last_updated:
-        title: Last updated date
+        title: Last Updated Date
```

Comment:
This will require a major version bump and a changelog entry to indicate the title change